test/Grpc.AspNetCore.Server.Tests/GrpcEndpointRouteBuilderExtensionsTests.cs

```diff
@@ -78,11 +78,11 @@ public void MapGrpcService_CanBind_CreatesEndpoints()
 
             var routeEndpoint1 = (RouteEndpoint)endpoints[0];
             Assert.AreEqual("/Greet.Greeter/SayHello", routeEndpoint1.RoutePattern.RawText);
-            Assert.AreEqual("POST", ((IHttpMethodMetadata)routeEndpoint1.Metadata.Single()).HttpMethods.Single());
+            Assert.AreEqual("POST", ((IHttpMethodMetadata)routeEndpoint1.Metadata.Single(m => m is HttpMethodMetadata)).HttpMethods.Single());
```

Comment:
@JamesNK Looks like this is due to some changes in endpoint routing. https://github.com/aspnet/AspNetCore/pull/6911 maybe?

Reply:
This is better:

```suggestion
            Assert.AreEqual("POST", routeEndpoint1.Metadata.GetMetadata<IHttpMethodMetadata>().HttpMethods.Single());
```

Out of interest, what is the other metadata?

Reply:
```
System.Runtime.CompilerServices.AsyncStateMachineAttribute
System.Diagnostics.DebuggerStepThroughAttribute
```